src/middleware/frontend-log-middleware.ts

```diff
@@ -50,14 +52,17 @@ export default (req: Request, _: Response, next: NextFunction): void => {
 		}
 	};
 
-	// Replaces req.log with default bunyan logger. So we can override Probot logger
-	req.log = getLogger(middlewareLoggerName);
-
-	const reqId = req.headers["x-request-id"] ||
-		req.headers["x-github-delivery"] ||
-		uuidv4();
-
-	req.addLogFields({ requestId: reqId });
-
-	next();
+	booleanFlag(BooleanFlags.PROPAGATE_REQUEST_ID, true).then(propagateRequestId => {
+		if (propagateRequestId) {
+			req.log = req.log.child({name: FILTERING_FRONTEND_HTTP_LOGS_MIDDLEWARE_NAME});
```

Comment:
Root of the PR: use Probot's logger instead of creating our own one, thus preserving requestId